src/gui/layout/qgslayoutviewtoolselect.cpp

```diff
@@ -154,7 +157,17 @@ void QgsLayoutViewToolSelect::layoutPressEvent( QgsLayoutViewMouseEvent *event )
     {
       selectedItem->setSelected( true );
     }
-    event->ignore();
+
+    // Due to the selection tolerance, items can be selected while the mouse is not over them,
+    // so we cannot just forward the mouse press event by calling event->ignore().
+    // We call startMove to simulate a mouse press event on the handles, then grabMouse
+    // to ensures that the mouse handles get the subsequent mouse move events.
+    mMouseHandles->startMove( view()->mapToScene( event->pos() ) );
+    if ( view()->scene()->mouseGrabberItem() != mMouseHandles )
+    {
+      mMouseHandles->grabMouse();
+    }
```

Comment:
could this be moved to QgsGraphicsViewMouseHandles::startMove ? it would be cleaner to keep all the grab/ungrab logic in that class alone.

Reply:
Sure.